src/components/field-text-tags-array/styled.ts

```diff
@@ -0,0 +1,152 @@
+import styled, { css } from 'styled-components';
+import { Field as FormikField } from 'formik';
+
+const Field = styled.div<{ error?: boolean }>`
+  display: flex;
+  flex-direction: column;
+  padding: 0 8px;
+
+  ${({ error }) =>
+    error &&
+    css`
+      & input {
+        border-color: red;
+
+        &:not(:disabled):focus {
+          box-shadow: 0 0 0 0.1rem rgba(255, 0, 0, 0.5);
+        }
+      }
+    `}
+`;
+
+const Label = styled.label`
+  font-weight: bold;
+  margin-bottom: 5px;
```

Comment:
I stedet for 'px' kan du vurdere 'em' for alle margins og paddings. Fordelen er at det generelt skalerer bedre med font-size, og font-size skal potensielt sett kunne endres av bruker i browser (rem).